gradio/interface.py

```diff
@@ -773,6 +769,39 @@ def integrate(self, comet_ml=None, wandb=None, mlflow=None) -> None:
             utils.integration_analytics(data)
 
 
+class TabbedInterface(Blocks):
+    def __init__(
+        self, interface_list: List[Interface], tab_names: Optional[List[str]] = None
+    ):
+        if tab_names is None:
+            tab_names = ["Tab {}".format(i) for i in range(len(interface_list))]
+        super().__init__()
+        with self:
+            with Tabs():
+                for (
+                    interface,
+                    tab_name,
+                ) in zip(interface_list, tab_names):
+                    with TabItem(label=tab_name):
+                        Interface(
+                            fn=interface.predict,
+                            inputs=interface.input_components,
+                            outputs=interface.output_components,
+                            examples=interface.examples,
+                            examples_per_page=interface.examples_per_page,
+                            live=interface.live,
+                            layout=interface.layout,
+                            interpretation=interface.interpretation,
+                            num_shap=interface.num_shap,
+                            title=interface.title,
+                            description=interface.description,
+                            article=interface.article,
+                            allow_flagging=interface.allow_flagging,
+                            flagging_options=interface.flagging_options,
+                            flagging_dir=interface.flagging_dir,
+                        )
+
+
```

Comment:
This seems not the right way to do this. Why can't you just call interface.render() instead of creating a new Interface? 
Also I'm a little skeptical we should even create TabbedInterface, instead of allowing users to naturally explore Blocks as they want more customizability.

Reply:
> Why can't you just call interface.render()

There's a few Interface kwargs that don't make sense to have within this context. For example, `theme`. So I just decided to pick the kwargs that make sense to include here.

> Also I'm a little skeptical we should even create TabbedInterface, instead of allowing users to naturally explore Blocks as they want more customizability

Open to discussion about this. We should keep in mind that we've heard directly from several users that they want something like this rather than having to delve into the lower-level Blocks API. E.g. https://github.com/gradio-app/gradio/issues/690#issuecomment-1050115607